docs/cmake-project-view.md

```diff
@@ -0,0 +1,124 @@
+            "projectStatusVisibility": "visible", "hidden" 
+        }
+}
+```
+
+* `folder` is the active folder in your workspace. This is where the project is scoped to.
+* `configure` is associated with the CMake Configure action. It can only be found in the Project Status View and can't be seen in the status bar, hence no `statusBarVisibility` property value
+* `configurePreset` is associated with the CMake Configure Preset. When selected, the user can modify their active CMake Configure Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project)
+* `kit` is associated with the active kit selected (when CMake Presets aren't present). When selected, the user can modify their active kit. To learn more about kits, please see [our kit documentation](https://github.com/microsoft/vscode-cmake-tools/docs/kits.md)
+* `variant` is associated with the active variant (when CMake Presets aren't present). When selected, the user can modify their active variant. Variant status does not show on the Project Status View, but will show in the status bar when set to `visible`. To learn more about variants, please see [our variant documentation](https://github.com/microsoft/vscode-cmake-tools/docs/variants.md)
+* `build` is associated with the CMake Build action. It invokes a CMake build on your build target using your build preset or variant.
+* `buildPreset` is associated with your active CMake Build Preset. When selected, the user can modify their active CMake Build Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project)
+* `buildTarget` is associated with your active CMake Build Target. When selected, the user can specify their active Build Target. This will be the target invoked when the user presses the `Build` icon in the status bar (if not hidden) or runs `CMake: Build Target` from the Command Palette
+* `ctest` is associated with running CTest. When selected, it will invoke CTest on the test preset specified. If there is no test preset specified, it will run all your tests by default.
+* `testPreset` is associated with selecting your active CMake Test Preset. When selected, the user can modify their active CMake Test Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project)
+* `launchTarget` is associated with selecting your active launch target. When selected, you can specify the active launch target.
+* `debug` is associated with the CMake Debug action. It invokes a debugger on the active launch target.
+* `launch` is associated with launching the target. It will run the specified target application in the terminal.
+
+To reset your CMake options back to the default, hover over your `CMake > Options: Status bar Visibility` and `CMake > Options: Advanced` settings and select the gear icons that appear for more options. From there, select `Reset Setting` on both options
```

Comment:
bar should be capitalized and the m in both CMakes need to be lowercase like Cmake. Also needs a period at the end.
Fixed:
To reset your CMake options back to the default, hover over your `Cmake > Options: Status Bar Visibility` and `Cmake > Options: Advanced` settings and select the gear icons that appear for more options. From there, select `Reset Setting` on both options.